pkgs/leak_tracker/lib/src/leak_tracking/_primitives/_retaining_path/_retaining_path.dart

```diff
@@ -7,7 +7,7 @@ import 'dart:developer';
 import 'dart:isolate';
 import 'package:vm_service/vm_service.dart' hide Isolate;
 
-/// Returns retaining path for an object, if it can be detected.
+/// Returns retainig path for an object, if it can be detected.
```

Comment:
`retainig` does not seem to be a word.
I googled it: https://www.google.com/search?q=retainig

Reply:
yes `retainig` was originally present in code, to resolve merge conflict I revert that and after again pushed the correct one.